### ISSUE TITLE
Feature request: allow to use NVIDIA DLSS for images

### ISSUE THREAD

**Issue by 3DRaven** [servo/webrender#4380]
Hi! NVIDIA have technology for upscale images by neural networks
https://developer.nvidia.com/dlss

This technology would allow us to send from server lower resolution image and upscale it on client side.

How about use this technology in webrender? It would be save some traffic for server and save power 
in mobile phones.

AMD have alternative https://www.amd.com/ru/technologies/radeon-software-fidelityfx

**Comment by kvark** [comment]
I don't think this is an option for WebRender at all. The page itself is not sent as an image, it's sent as a display list, so DLSS isn't helping. The images to be used for the page have to be precise in order for the Web to be portable, so we can't apply random compression schemes like this to reduce the transfer cost.
